Mixpanel/DecideRequest.swift

```diff
@@ -40,10 +40,10 @@ class DecideRequest: Network {
 
     func sendRequest(distinctId: String,
                      token: String,
-                     completion: (DecideResult?) -> Void) {
+                     completion: @escaping (DecideResult?) -> Void) {
```

Comment:
The @escaping means that the completion closure could still run after the function has ended. Because the decide request is sent asynchronously to the server, the completion will escape the scope of the method